pkg/influx/recorder_test.go

```diff
@@ -92,7 +94,7 @@ influxdb_proxy_ingester_data_conversion_seconds_count 1
 		},
 		"Register version build timestamp": {
 			measure: func(r Recorder) {
-				_ = r.RegisterVersionBuildTimestamp()
+				_ = r.RegisterVersionBuildTimestamp(log.NewLogfmtLogger(log.NewSyncWriter(os.Stdout)))
```

Comment:
for tests we usually use a nooplogger so as not to spam the console

Reply:
Good to know - I'll make the change!